.projenrc.ts

```diff
@@ -85,7 +84,7 @@ const project = new CustomTypescriptProject({
     "fs-extra",
     "minimatch",
     "@functionless/nodejs-closure-serializer",
-    "@functionless/ast-reflection@^0.1.1",
+    "@functionless/ast-reflection@file:../ast-reflection",
```

Comment:
mean to keep this?

Reply:
Nope, never do. Wonder if we can have an eslint rule lol